File: 20. Valid Parentheses/20.py
class Solution:
    def isValid(self, s: str) -> bool:
        if (len(s) <= 1):
            return False
   
        for j in range(10001):
            substr = ["{}", "[]", "()"]
            for i in substr:
                if i in s:
                    s = s.replace(i, "")

        if (len(s) > 0):
            return False
        
        return True
        

File: 20. Valid Parentheses/20.php
<?php

class Solution {

    /**
     * @param String $s
     * @return Boolean
     */
    function isValid($s) {
        if (strlen($s) <= 1) {
            return false;
        }
    
        for ($j = 0; $j <= 10000; $j++) {
            $substr = ["{}", "[]", "()"];
            for ($i = 0; $i < count($substr); $i++) {
                if (str_contains($s, $substr[$i])) {
                    $s = str_replace($substr[$i], "", $s);
                }
            }
            
        }
        
        if (strlen($s) > 0) {
            return false;
        }
        return true;
    }
}